obsidian.css

```diff
@@ -752,10 +788,6 @@ settings:
         description: Move the editor header hastags # to above the header instead.
         id: hash-top
         type: class-toggle
-    -
```

Comment:
I recommend adding back this class toggle

Reply:
Re-added.